pkg/skaffold/build/kaniko/secret.go

```diff
@@ -124,3 +124,53 @@ func (b *Builder) setupDockerConfigSecret(out io.Writer) (func(), error) {
 		}
 	}, nil
 }
+
+func (b *Builder) setupAWSSecretSecret(out io.Writer) (func(), error) {
```

Comment:
What is specific to AWS about this? Looks like the only thing thats different is the filename in the secret, which should already be configurable through the YAML (if not, I think thats the feature that we'd want instead of a separate code path).

Reply:
Thanks for the feedback. That makes sense.

Is something like this what you had in mind?

```
kaniko:
  # ...
  secrets:
    - path: <secret1-path>
      name: <secret1-name>
    - path: <secret2-path>
      name: <secret2-name>
```

If so I can give this a try in the next few days.